bridge/engine/ActiveMediaList.h

```diff
@@ -223,6 +227,7 @@ class ActiveMediaList
     void updateLevels(const uint64_t timestampMs);
     void updateActiveAudioList(size_t endpointIdHash);
     bool updateActiveVideoList(const size_t endpointIdHash);
+    std::atomic_bool _c9_conference;
```

Comment:
We do not need this do we?

Reply:
I think we need some flag to indicate how to proceed with regards of active talker detection. At any point of time, if the request comes to get a set of active talkers, we'll check the flag, and either use the limited set of endpoints that have `isPtt` flag, _or_ we use our own ranking.